deploy/smcp-templates/product

```diff
@@ -0,0 +1,37 @@
+apiVersion: maistra.io/v1
```

Comment:
nit: should the name of the template be "service-mesh", not "product"?

Reply:
Done